core/src/services/fs/backend.rs

```diff
@@ -441,6 +441,17 @@ impl Accessor for FsBackend {
         Ok((RpWrite::new(), FsWriter::new(target_path, tmp_path, f)))
     }
 
+    async fn copy(&self, from: &str, to: &str, _args: OpCopy) -> Result<RpCopy> {
```

Comment:
Some operations may behave differently between our different backends. 

For example, for POSIX copy, if the parent path to does not exist, it will report an error, while s3 will use the normal write for our subsequent implementation because it does not provide a copy operation, at which point it will automatically create the path. 

So the question is, do we need to standardize the behaviour of these operations, e.g., create all the parent paths automatically?

Reply:
Yes, following the same behaviour like write does.

Reply:
Make sense.